marklogic-data-hub-central/ui/src/config/hub-properties.config.js

```diff
@@ -30,6 +30,22 @@ const hubPropertiesConfig = [
     propertyPath: 'createdByStep',
     entityTypeId: ' '
   },
+  {
+    facetName: "sourceName",
+    displayName: "SourceName",
```

Comment:
Can there be spaces in a "displayName"? We'd want this to be as user-friendly as possible, so "Source Name" would be better than "SourceName".

Reply:
Yes there can be spaces, thanks for noticing that.